seqlearn/hmm.py

```diff
@@ -68,10 +68,10 @@ def fit(self, X, y, lengths):
         final_prob -= logsumexp(final_prob)
```

Comment:
Another observation: final_prob is the same as init_prob here. Should this be `Y[end - 1]` instead?